src/react-components/hub-create-panel.js

```diff
@@ -214,6 +226,22 @@ class HubCreatePanel extends Component {
                             <span>{environmentAuthor.name}</span>
                           </span>
                         ))}
+                      {environmentAuthor &&
+                        environmentAuthor.organization &&
+                        (environmentAuthor.organization.url ? (
+                          <a
+                            href={environmentAuthor.organization.url}
+                            target="_blank"
+                            rel="noopener noreferrer"
+                            className="create-panel__form__environment__picker__labels__header__org"
```

Comment:
the color of this IMO should be a bit darker (`#bbb` or one of the [SASS colour variables](https://github.com/mozilla/hubs/blob/c6d45ceb/src/assets/stylesheets/shared.scss#L1-L7) or `lighten`/`darken` variations thereof)